src/pages/plugins/index.js

```diff
@@ -104,28 +104,35 @@ function PluginComponent({ plugin }) {
   return (
     <div className="row padding-vert--md">
       <div className="col col--10 col--offset-1">
-        <div key={plugin.key} className={ plugin.deprecatedMessage ? `card ${classnames('card__deprecated', styles.pluginDeprecatedCard)}` : "card"}>
-          {!plugin.deprecatedMessage || (
+        {plugin.deprecatedMessage ? (
+          <div key={plugin.key} className={`card ${classnames('card__deprecated', styles.pluginDeprecatedCard)}`}>
```

Comment:
For example, you can use the classnames module like so:
```jsx
const cardClasses = classnames('card', {
    [styles.pluginDeprecatedCard]: plugin.deprecatedMessage ? true : false
});
```

And then you can use it like:
```jsx
<div key={plugin.key} className={cardClasses}>
```

@donghwipark It seems like you didn't understand the API for classnames because you included `card__deprecated` which is never defined or used anywhere. Make sure you review https://www.npmjs.com/package/classnames

Reply:
I implemented as written, but but still has issue with understanding about classnames especially with styles.
On deprecated card 
`{ [styles.pluginDeprecatedCard]: plugin.deprecatedMessage ? true : false }`
Does this will be the 
`{ [styles.pluginDeprecatedCard]: true }`
And this will trigger the pluginDeprecatedCard styles?